neard/build.rs

```diff
@@ -8,27 +8,58 @@
 
 use anyhow::{anyhow, Result};
 
+/// Returns Cargo package version.
+///
+/// This simply reads `CARGO_PKG_VERSION` environment variable but also
+/// translates version ‘0.0.0’ into ‘trunk’.
+fn get_package_version() -> Result<String> {
+    env("CARGO_PKG_VERSION").and_then(|ver| match ver.to_string_lossy() {
+        std::borrow::Cow::Borrowed("0.0.0") => Ok("trunk".to_string()),
+        std::borrow::Cow::Borrowed(version) => Ok(version.to_string()),
+        std::borrow::Cow::Owned(version) => {
+            Err(anyhow!("invalid ‘CARGO_PKG_VERSION’: {}", version))
+        }
+    })
+}
+
 /// Returns value of given environment variable or error if missing.
+///
+/// This also outputs necessary ‘cargo:rerun-if-env-changed’ tag to make sure
+/// build script is rerun if the environment variable changes.
 fn env(key: &str) -> Result<std::ffi::OsString> {
     println!("cargo:rerun-if-env-changed={}", key);
     std::env::var_os(key).ok_or_else(|| anyhow!("missing ‘{}’ environment variable", key))
 }
 
+/// Returns value of given environment variable or given default value.
+///
+/// This also outputs necessary ‘cargo:rerun-if-env-changed’ tag to make sure
+/// build script is rerun if the environment variable changes.
+fn env_or(key: &str, default: impl std::convert::Into<std::ffi::OsString>) -> std::ffi::OsString {
```

Comment:
```suggestion
fn env_or(key: &str, default: impl Into<std::ffi::OsString>) -> std::ffi::OsString {
```

`Into` in in prelude